README.md

```diff
@@ -1,52 +1,86 @@
-Yunohost app for « I hate money » budget web app
-================================================
+# I Hate Money for YunoHost
 
-[![Install « I hate money » with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=ihatemoney)
+[![Integration level](https://dash.yunohost.org/integration/ihatemoney.svg)](https://dash.yunohost.org/appci/app/ihatemoney) ![](https://ci-apps.yunohost.org/ci/badges/ihatemoney.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/ihatemoney.maintain.svg)  
+[![Install ihatemoney with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=ihatemoney)
 
-- Supported Yunohost versions : 2.6.x, 2.7.x 3.x
-- Tested Yunohost version : 3.3.1
+*[Lire ce readme en français.](./README_fr.md)*
 
-*NB: That means I'll try not to drop support for YunoHost 2.x too soon, and
-accept patches to keep retro-compatibility, but I'll not test it myself against
-YunoHost 2.x*
+> *This package allows you to install I Hate Money quickly and simply on a YunoHost server.  
+If you don't have YunoHost, please consult [the guide](https://yunohost.org/#/install) to learn how to install it.*
+
+## Overview
+I Hate Money is a web application made to ease shared budget management. It keeps track of who bought what, when, and for whom; and helps to settle the bills.
+
+**Shipped version:** 4.1.3
+
+## Screenshots
+
+![](sources/Screenshots.png)
+
+## Demo
+
+* [Official demo](https://ihatemoney.org/)
+
+## Configuration
+
+How to configure this app: From an admin panel, a plain file with SSH, or any other way.
+
+## Documentation
+
+ * Official documentation: Link to the official documentation of this app
+ * YunoHost documentation: If specific documentation is needed, feel free to contribute.
+
+## YunoHost specific features
 
 Backs on MySQL database, the identifiers are per-project, not per-user, so no
-way to do advanced SSO integration with yunohost accounts.
+way to do advanced SSO integration with YunoHost accounts.
 
 The behaviour is either:
 
 - **non-public app**:
-  - yunohost login required
+  - YunoHost login required
   - per-project identifiers required
-  - any yunohost user with authorized access to the app can create a new
+  - any YunoHost user with authorized access to the app can create a new
     project).
 - **public app** :
-  - no yunohost login required
+  - no YunoHost login required
   - per-project identifiers required
   - any visitor can create a new project.
 
-Update
-------
+#### Multi-user support
```

Comment:
Either remove this section or move the comment about per-project identifiers here ?